share/spice/forecast/forecast.js

```diff
@@ -158,10 +151,9 @@
         dailyObj[i] = days[i];
         day = days[i];
 
-        tmp_date = new Date();
-        tmp_date.setDate(date_i+i);
-        dailyObj[i].date = tmp_date.toDateString().substr(4,6);
-        dailyObj[i].day = i == 0 ? 'Today' : day_strs[(today_i+i)%7];
+        tmp_date = moment(today);
+        tmp_date.add(i, 'days');
+        dailyObj[i].date = tmp_date.format("MMM D");        
```

Comment:
Can we simplify this to `moment(today).add(1, 'days').format("MMM D")`? What do you think? It seems simpler.